org.omg.sysml.jupyter.kernel/build.gradle

```diff
@@ -66,23 +81,21 @@ artifacts {
     archives shadowJar
 }
 
+generateKernelJson.dependsOn shadowJar
 zipKernel.dependsOn shadowJar
 installKernel.dependsOn shadowJar
 
-installKernel {
-    kernelInstallPath = commandLineSpecifiedPath(userInstallPath)
+installKernel{
+    kernelInstallPath = commandLineSpecifiedPathOr(userInstallPath)
 }
 
+
 zipKernel {
     from('src/main/resources/installer/') {
         into '/'
     }
-    from('../') {
```

Comment:
I'm fixing this one too. Sorry for this.

Reply:
@seidewitz, @himi I've corrected the mistake. Those accidental changes should not show up anymore